### DISCUSSION TITLE
Design > components screen unresponsive

### DISCUSSION THREAD

**Discussion by chrismb23** [Budibase/budibase#8492] [Help]
Hi team,

As of a few days I'm experiencing that the Design > Components screen is unresponsive for some screens. When this happens I can still click on the individual components on the left, but the middel and right parts of the scree seem frozen and do not refresh based on my clicks. I also see the URL changing when clicking but nothing happens. The only way to get out of this freeze is by reloading the page. 

Is this known behaviour or is there something I can do to prevent his?

Thanks!

**Comment by chrismb23** [comment]
Not sure whether it’s relevant but I did eject a form block on an auto-generated screen before this happened. I seem to be able to reproduce this if I perform the same steps with a new screen.

**Reply by andz-bb** [reply]
@chrismb23 I think the issue you're having with form block ejection has just been logged here: #8493. There's a PR with a fix here: #8489 - hopefully it will be released very soon!

**Reply by chrismb23** [reply]
That’s exactly what’s happening indeed. Thanks!

**Comment by andz-bb** [comment]
hey @chrismb23 - could you send an app export to support@budibase.com or [on our discord server](https://discord.gg/E3YXNAg4m5)? I can take a closer look and see if I can replicate the issue on my end.
